src/app/shared/localized-currency-pipe/localized-currency.pipe.ts

```diff
@@ -25,8 +25,10 @@ export class LocalizedCurrencyPipe implements PipeTransform, OnDestroy {
     });
   }
 
-  transform(value: any): any {
-    return `${value} ${LOCALIZED_CURRENCY[this.lang]}`;
+  transform(value: any, fixedDigits: boolean = false): any {
```

Comment:
The value parameter is directly converted to a number, which might runtime errors if the input is invalid (e.g., null, undefined):
```  
if (value == null || isNaN(+value)) 
```